src/content/whats-new/2022/03/whats-new-03-01-Kubernetes-APM.md

```diff
@@ -0,0 +1,22 @@
+---
+title: 'Minimize Kubernetes complexity to build performant applications'
+summary: 'Analyze application and Kubernetes observability data in one place'
+releaseDate: '2022-03-01'
+learnMoreLink: 'https://newrelic.com/blog/nerdlog/kubernetes-experience-in-APM'
+---
+## See Kubernetes details in APM
+
+Use our new UI that shows how cluster performance impacts your applications.
+
+* Analyze cluster performance in a single, curated UI.
+* Get alerted with a real-time activity stream for Kubernetes events and critical issues.
+* Correlate performance anomalies with one-click logs and side-by-side (selectable) metrics.
+
+## How to get started
+Go to **[one.newrelic.com](http://one.newrelic.com) > Explorer > Services - APM > (select an app) > Kubernetes**. To see Kubernetes performance data in New Relic One, do the following:
+
+* Install the New Relic [infrastructure integration](https://docs.newrelic.com/docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-using-helm/).
+* See our documentation about [New Relic APM](https://docs.newrelic.com/docs/apm/new-relic-apm/getting-started/introduction-apm/), as well as information about the [Kubernetes summary page](https://docs.newrelic.com/docs/apm/apm-ui-pages/monitoring/kubernetes-summary-page).
```

Comment:
We no longer use "New Relic APM" together.

Reply:
@urbiz-nr, how about this revision:

```
* See our general [APM](https://docs.newrelic.com/docs/apm/new-relic-apm/getting-started/introduction-apm/) documentation, as well as information about the [Kubernetes summary page](https://docs.newrelic.com/docs/apm/apm-ui-pages/monitoring/kubernetes-summary-page).
```